AzurePSDrive.psd1

```diff
@@ -8,7 +8,7 @@
 RootModule = '.\AzurePSDrive.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.6.0'
+ModuleVersion = '0.7.0'
```

Comment:
Please publish the update to PSGallery, so that the next PSCloudShell build picks this up